### PR TITLE
[NPU] add npu kernel for softmax_with_cross_entropy

### PR DESCRIPTION
**Pull request by zhiqiu**
### PR types

New features
### PR changes

OPs
### Describe

[NPU] softmax with cross entropy
image
image
image

### CODE REVIEW

**Review by liym27 — APPROVED** [review]
LGTM

**Review by frankwhzhang — APPROVED** [review]
LGTM
